superset-frontend/src/explore/components/ExploreViewContainer/index.jsx

```diff
@@ -191,36 +193,38 @@ function ExploreViewContainer(props) {
   };
 
   const addHistory = useCallback(
-    ({ isReplace = false, title } = {}) => {
+    async ({ isReplace = false, title } = {}) => {
       const formData = props.dashboardId
         ? {
             ...props.form_data,
             dashboardId: props.dashboardId,
           }
         : props.form_data;
       const payload = { ...formData };
-      const longUrl = getExploreLongUrl(
-        formData,
-        props.standalone ? URL_PARAMS.standalone.name : null,
-        false,
-        {},
-        props.force,
-      );
+      const chartId = formData.slice_id || 0;
```

Comment:
very minor thought, but I'd prefer to leave it `undefined` if the chart hasn't yet been saved.